stories/theme/ag/small/Listings/MobileGridListing.css

```diff
@@ -0,0 +1,199 @@
+.ListingCell {
+  border-radius: 5px;
+  border: 1px solid lightgrey;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.MobileGridListing {
+  width: 100%;
+  height: 240px;
+  margin-bottom: 6px;
+}
+
+.MobileGridListing_Bottom,
+.MobileGridListing_BedsAndCta {
+  width: 100%;
+}
+
+.MobileGridListing_Top {
+  height: 160px;
+  width: 100%;
+}
+
+.MobileGridListing_Top .PhotoCarousel :global .image-gallery :global .image-gallery-slide img{
+  height: 160px;
+  width: 100%;
+}
+
```

Comment:
Everything from this point on is redundant of https://github.com/rentpath/react-ui/blob/master/stories/theme/ag/small/MobileMapListing.css.  That's 170 extra lines that shouldn't be needed to make this happen.